paasta_tools/utils.py

```diff
@@ -1273,39 +1390,66 @@ def format_path(self, service: str, component: str, level: str, cluster: str, in
             instance=instance,
         )
 
-    def log(
-        self,
-        service: str,
-        line: str,
-        component: str,
-        level: str = DEFAULT_LOGLEVEL,
-        cluster: str = ANY_CLUSTER,
-        instance: str = ANY_INSTANCE,
-    ) -> None:
-        path = self.format_path(service, component, level, cluster, instance)
-
+    def _log_message(self, path: str, message: str) -> None:
         # We use io.FileIO here because it guarantees that write() is implemented with a single write syscall,
         # and on Linux, writes to O_APPEND files with a single write syscall are atomic.
         #
         # https://docs.python.org/2/library/io.html#io.FileIO
         # http://article.gmane.org/gmane.linux.kernel/43445
 
-        to_write = "{}{}".format(
-            format_log_line(level, cluster, service, instance, component, line),
-            self.line_delimeter,
-        )
-
         try:
             with io.FileIO(path, mode=self.mode, closefd=True) as f:
                 with self.maybe_flock(f):
                     # remove type ignore comment below once https://github.com/python/typeshed/pull/1541 is merged.
```

Comment:
While you're here, do you mind removing this line and the `# type: ignore` comment below?